transports/upgrades/sqlserver-31to4.md

```diff
@@ -18,3 +18,15 @@ The Transaction scope [transaction mode](/transports/sql/transactions.md) is not
 To run the upgraded project on .NET Core the transport needs to be switched to one of the native transactions modes. Consider using the [Outbox](/nservicebus/outbox) to maintain the same *exactly-once processing* guarantees.
 
 NOTE: Transaction scope is supposed to be supported by `SqlConnection` in future versions of .NET Core. 
+
+## Multi-instance mode
+
+The multi-instance mode has been deprecated in Version 4. NServiceBus topologies with queues distributed between multiple catalogs and SQL Server instances can be migrated using a combination of [Transport Bridge](/nservicebus/bridge/) and multi-catalog [addressing](/transports/sql/addressing.md):
```

Comment:
Some of this part would be good as an intro to the sample :)

Reply:
Done.